.github/workflows/continuous.yaml

```diff
@@ -30,26 +30,20 @@ jobs:
       matrix:
         app: [ web, node ]
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
+        uses: docker/setup-qemu-action@v3
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
+        uses: docker/setup-buildx-action@v3
       - id: auth
         name: Authenticate to Google Cloud
-        uses: google-github-actions/auth@v1
+        uses: google-github-actions/auth@v2
         with:
           token_format: 'access_token'
           workload_identity_provider: 'projects/${{ secrets.DEV_GKE_PROJECT_ID}}/locations/global/workloadIdentityPools/github/providers/github'
           service_account: '${{ secrets.DEV_GKE_SA }}'
-      - name: Login to GCR
```

Comment:
I take it this step is no longer needed (and in other removed places)?

Reply:
yes, it's a login to the old container registry which we've replaced with the artifact registry - we removed the push to this registry some time ago, I see the login was not taken out at the same time